SIT/regression/03_performance/migrate.properties

```diff
@@ -6,3 +6,6 @@ spark.cdm.schema.target.keyspaceTable             target.regression_performance
 
 spark.cdm.perfops.numParts                        32
 spark.cdm.perfops.batchSize                       1
+
+spark.cdm.perfops.printStatsAfter                 450
```

Comment:
LGTM. Let's please coordinate with docs crew to publish this at https://docs.datastax.com/en/astra-serverless/docs/migrate/cassandra-data-migrator.html page